static/app/components/eventTitleTreeLabel.tsx

```diff
@@ -33,7 +36,7 @@ function EventTitleTreeLabel({treeLabel}: Props) {
             return (
               <Fragment key={index}>
                 <Divider>{'|'}</Divider>
-                {label}
+                {label?.function || label?.package || label?.type}
```

Comment:
same here

Reply:
we should always have a value here, otherwise we will be rendering an empty element